pytradfri/resource.py

```diff
@@ -42,11 +43,17 @@ def observe_callback(value):
 
             Returns a Command.
             """
-            self.raw = value
-            callback(self)
+            if self.raw != value:
+                self.raw = value
+                callback(self)
 
+        def error_callback(value):
+            print(value)
```

Comment:
If the caller is interested in logging, it's their responsibility, not this library.

Reply:
Amended in cd9f6866456f333a7e3043eb55ef7944d8961292